.github/ISSUE_TEMPLATE/1-PUB_SITE.md

```diff
@@ -0,0 +1,13 @@
+---
+name: There is a problem with pub.dartlang.org
+about: A `pub get` fails because of an outage at `pub.dartlang.org`, the site
+  cannot be loaded, or there is an issue with a particular package such as a
+  request to have a published package removed.
+
+---
+
+**********************************************************
+**This is not the repository you want to file issues in!**
+**********************************************************
+
+For any issues with `pub.dartlang.org` file issues at https://github.com/dart-lang/pub-dartlang-dart/issues
```

Comment:
"For any issues with the `Pub.dev` repository, file..."

Reply:
Done